tx-source/site_core.php

```diff
@@ -597,6 +598,7 @@
 $Definition['Enter your search term.'] = 'Enter your search term.';
 $Definition['Erase User Content'] = 'Erase User Content';
 $Definition['Enter your username to continue.'] = 'Enter your username to continue.';
+$Definition['error'] = 'error';
```

Comment:
I think we should just use strtolower() on the existing translation since it costs money to get these things translated.